scripts/ci/run-benchmark

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+"""Script to benchmark several high level cli commands.
+
+As of now this benchmarks `cp` and `rm` with test cases for multiple 4kb files
+(default 10000 files) and a single large file (default 10gb, `cp` only).
+"""
+import os
+from subprocess import check_call
+from datetime import datetime
+import random
+import argparse
+
+TEST_BUCKET = os.environ.get('PERF_TEST_BUCKET')
+MANY_FILES_DIR = "files/10k"
+LARGE_FILE_DIR = "files/10gb"
+
+
+def run(command):
+    return check_call(command, shell=True)
+
+
+def generate_run_id():
+    run_id = datetime.now().strftime("%Y-%m-%d-%H-%M-%S-")
+    run_id += str(random.randint(1, 10000))
+    return run_id
+
+
+def initialize_files(num_files, file_size):
+    if not os.path.exists(MANY_FILES_DIR):
+        os.makedirs(MANY_FILES_DIR)
+        run('caf gen --file-size 4kb --max-files %s --directory %s' %
+            (num_files, MANY_FILES_DIR))
+    if not os.path.exists(LARGE_FILE_DIR):
+        os.makedirs(LARGE_FILE_DIR)
+        run('caf gen --file-size %s --max-files 1 --directory %s' %
+            (file_size, LARGE_FILE_DIR))
+
+
+def main(args):
+    initialize_files(args.num_files, args.large_file_size)
+    run_id = generate_run_id()
+    results_dir = os.path.join('results', run_id)
+    os.makedirs(results_dir)
+    benchmark(args.bucket, results_dir, args.num_iterations)
+    print("RUN ID: " + run_id)
+
+
+def benchmark(bucket, results_dir, num_iterations=1):
+    perf_dir = os.path.dirname(os.path.realpath(__file__))
+    perf_dir = os.path.join(perf_dir, os.pardir, 'performance')
```

Comment:
You can also just call `os.path.dirname()` twice instead of using `pardir`:

```
perf_dir = os.path.join(
    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
    'performance')
```